syncer/error.go

```diff
@@ -95,10 +96,7 @@ func originError(err error) error {
 
 // handleSpecialDDLError handles special errors for DDL execution.
 func (s *Syncer) handleSpecialDDLError(tctx *tcontext.Context, err error, ddls []string, index int, conn *DBConn) error {
-	parser2, err2 := s.fromDB.getParser(s.cfg.EnableANSIQuotes)
-	if err2 != nil {
-		return err // return the original error
-	}
+	parser2 := parser.New()
```

Comment:
add a comment to explain why can we use the default parser here?

Reply:
https://github.com/pingcap/dm/pull/889/commits/6498e2452c72aae9ac711e184c589d3ffc7166af